musicpalette/musicpalette/PlaylistTableViewController.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  PlaylistTableViewController.swift
+//  musicpalette
+//
+//  Created by Lilly Liu on 11/4/20.
+//  Copyright © 2020 musicpalette. All rights reserved.
+//
+
+import UIKit
+
+class PlaylistTableViewController: UITableViewController {
+    var swipeModels: [SwipeSelectorModel] = []
+
+        init(swipeModels: [SwipeSelectorModel]) {
+            self.swipeModels = swipeModels
+            super.init(nibName: nil, bundle: nil)
+        }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+        //MARK: UITableViewDataSource
+
+        override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+            return 0
+        }
+
+        // Provide a cell object for each row.
+        override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+            //TODO: Add Custom TableViewCell to display the cell
+            return UITableViewCell()
+        }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        // Uncomment the following line to preserve selection between presentations
+        // self.clearsSelectionOnViewWillAppear = false
+
+        // Uncomment the following line to display an Edit button in the navigation bar for this view controller.
+        // self.navigationItem.rightBarButtonItem = self.editButtonItem
+    }
+
+    // MARK: - Table view data source
+
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        // #warning Incomplete implementation, return the number of sections
+        return 0
+    }
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
```

Comment:
looks like compiler is complaining since you have duplicated functions, please remove